### PR TITLE
fix: [CVS-9208]: Update Changes

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
retrigger jest

**Review by tan-nhu — APPROVED** [review]

**Review by vardanbansal-harness — DISMISSED** [review]

File: src/modules/85-cv/pages/changes/CVChanges.tsx [hunk 1]
```diff
@@ -44,7 +47,12 @@ export const CVChanges = ({ updateTime }: { updateTime?: Date }): JSX.Element =>
   const { environmentOptions } = useGetHarnessEnvironments()
   const { getString } = useStrings()
   const sourceTypes = useMemo(() => {
-    return getChangeSourceOptions(getString)
+    return getChangeSourceOptions(getString).map((changeSourceOption: SelectOption) => {
+      if (changeSourceOption?.value === ChangeSourceCategoryName.ALERT) {
```
Comment (anjanBalgovind) [inline src/modules/85-cv/pages/changes/CVChanges.tsx:51]: change this in the options list itself. No need to do it here

Comment (vasuki-kalluri) [reply src/modules/85-cv/pages/changes/CVChanges.tsx:51]: options list if Global right. The change should only be for Changes page.

Comment (vasuki-kalluri) [reply src/modules/85-cv/pages/changes/CVChanges.tsx:51]: It is used at multiple places. The change is supposed to be only for Changes page.

**Review by anjanBalgovind — DISMISSED** [review]

**Comment by vasuki-kalluri** [comment]
retrigger cypress

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Comment by vasuki-kalluri** [comment]
trigger cypress

**Review by diptiman-adak — APPROVED** [review]
Approving as per request over the slack

**Pull request merged**